### PR TITLE
Add restart instructions to jobrunner playbook

### PR DESCRIPTION
**Pull request by lucyb**
This expands on the existing section to include planned maintenance and what to do after a restart.

### CODE REVIEW

File: jobrunner/playbook.md [hunk 1]
```diff
@@ -211,3 +221,26 @@ The next time job-runner restarts (which should be after the reboot) it
...
 user should not have to do anything.
 
+### Planned maintenance
+
+Sometimes we are informed that a reboot will take place out of hours. In this case, in order to ensure a graceful shutdown and to avoid someone having to work late, the [preparing for reboot](#preparing-for-reboot) section can be run as a single command with a sleep statement.
+
+For example, this will start shutting things down in four hours:
+
+```sh
+sleep 4*3600; systemctl stop job-runner; python -m jobrunner.cli.prepare_for_reboot
```
Comment (bloodearnest) [inline jobrunner/playbook.md:231]:
```suggestion
sleep $((4*3600)); systemctl stop job-runner && python -m jobrunner.cli.prepare_for_reboot
```

File: jobrunner/playbook.md [hunk 2]
```diff
@@ -211,3 +221,26 @@ The next time job-runner restarts (which should be after the reboot) it
...
+```
+
+### After a restart
+
+#### Syncing files to Level 3
+
+This command ensures files are copied to Level 3. It's only used by a small number of researchers, as normally researchers would access their outputs via Level 4.
```
Comment (bloodearnest) [inline jobrunner/playbook.md:238]:
```suggestion
This command ensures files manually generated on the Windows host are copied to Level 3. It's only used by a small number of researchers, as normally researchers would access their outputs via Level 4.
```

**Review by bloodearnest — APPROVED** [review]

File: jobrunner/playbook.md (hunk 2 above)
Comment (lucyb) [reply jobrunner/playbook.md:238]: Thank you! I've been using Level 3 to refer to the Windows host. You seem to be using it here to refer to the Ubuntu VM? What is the correct terminology?

Comment (bloodearnest) [reply jobrunner/playbook.md:238]: Both are level 3 in terms of access level - they both have access to the DB, and to the sensitive level 3 files (although those are actually stored on the VM).  I've been using "Windows host" and "Ubuntu VM" do distinguish between the two.

Comment (lucyb) [reply jobrunner/playbook.md]: Okay, that makes sense. How about the change I've just pushed? Presumably the files are picked up by L4, so maybe I should make that clearer?

**Review by bloodearnest — APPROVED** [review]

**Pull request merged**